packages/app/src/types.ts

```diff
@@ -1,3 +1,12 @@
 export enum Pages {
   'home' = '/',
 }
+
+/**
+ * This should be generated automatic with Codegen and GraphQL endpoints
```

Comment:
The accounts and all data are going to be saved locally on the browser. I think the idea of having the types is not temp but permanent.